the-british-journal-of-sociology.csl

```diff
@@ -167,9 +167,9 @@
           </group>
         </else-if>
         <else>
-          <group suffix="', ">
-            <text macro="title" prefix=" '"/>
-            <text macro="editor" prefix=" "/>
+          <group prefix=" '" suffix="', " delimiter=" ">
```

Comment:
this won't typically actually occur, but obviously we don't want quotes around the editor.
I'd like the quotes="true" to be handled to the title macro (for the else case in l. 83) and then removed from the rest of the style.